### PR TITLE
update to StarAlgebras.jl-0.3

### PR DESCRIPTION
**Pull request by kalmarek**
This was less painful than expected ;)

CC: @blegat

### CODE REVIEW

File: src/wedderburn_decomposition.jl [hunk 1]
```diff
@@ -173,15 +173,15 @@ end
...
 end
 
 function invariant_vectors(
     G::Group,
     act::ByPermutations,
-    basis::StarAlgebras.Basis{T,I},
+    basis::StarAlgebras.ExplicitBasis{T,I},
 ) where {T,I}
     tovisit = trues(length(basis))
     invariant_vs = Vector{SparseVector{Rational{Int},I}}()
```
Comment (blegat) [inline src/wedderburn_decomposition.jl:187]: The line below fails because `eachindex(basis)` is not defined for `FixedBasis`

Comment (blegat) [reply src/wedderburn_decomposition.jl:187]: It was broken by https://github.com/JuliaAlgebra/StarAlgebras.jl/pull/21/commits/7f17590e35ec15d0748bb2dd4972f122e06f43e8 I believe

**Comment by kalmarek** [comment]
@blegat this one is +250 - 200; the other pull (#69) is a bit larger, but contains almost all of this one ;)

**Comment by blegat** [comment]
Merging this one first was the right call

**Pull request merged**